website/docs/language/settings/backends/s3.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-sidebar_label: s3
+sidebar_label: Amazon S3
```

Comment:
I would prefer to keep this as just "s3" if possible as we may in the future try and support other s3-compatible backends officially.